x-pack/test/functional/apps/infra/link_to.ts

```diff
@@ -22,7 +22,7 @@ export default ({ getPageObjects, getService }: FtrProviderContext) => {
         state: undefined,
       };
       const expectedSearchString =
-        "logFilter=(expression:'trace.id:433b4651687e18be2c6c8e3b11f53d09',kind:kuery)&logPosition=(position:(tiebreaker:0,time:1565707203194))&sourceId=default&_g=()";
+        "logFilter=(expression:'trace.id:433b4651687e18be2c6c8e3b11f53d09',kind:kuery)&logPosition=(position:(tiebreaker:0,time:1565707203194))&sourceId=default";
```

Comment:
Is this change expected?

Reply:
It is insofar as me making it, but I don't understand the change in behaviour. This functional test is testing `pages/link_to/redirect_to_logs`, in that component we only add the expected query parameters (`logPosition` etc). 

I know `_g=()` is something added globally by Kibaba, but I don't really understand _when_. We don't read from or write to `_g=()` in our apps.  Within this component we are now within our app, and our react-router context. I guess the old behaviour was `&_g=()` was set at some point in the global angular router, and then our stuff just merged with it. And now it's just the stuff we actually set ourselves.

Ultimately, I don't know where the difference in behaviour comes from. As I haven't changed the uses of `ui/utils/query_string` or anything to do with URLs.

However, this new behaviour is more inline with what I'd expect (the URL only contains what we set and use). Would be nice to understand the root change, though.

Reply:
Oh - actually, as a guess, maybe it's because we removed all that code that accessed Angular and did those buffer calls, and we're just mounting straight within the "React world" now.

Reply:
Ohhh that's a good guess, and I like the new behavior too. Sounds safe.